packages/datetime2/test/components/dateRangePicker3Tests.tsx

```diff
@@ -1332,6 +1332,33 @@ describe("<DateRangePicker3>", () => {
         });
     });
 
+    describe("outside days", () => {
```

Comment:
these tests are very closely tied to the implementation but figured it was good to get something in testing the behavior so this doesn't unintentionally regress